x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/index_properties/helpers.ts

```diff
@@ -40,19 +41,21 @@ export const getSortedPartitionedFieldMetadata = ({
   mappingsProperties,
   unallowedValues,
 }: {
-  ecsMetadata: Record<string, EcsMetadata> | null;
```

Comment:
For reviewers:

ecsMetadata represents EcsFlat object which is imported statically so it can never be null.